commit 6f006fa560eef1d6fe2d50f00e8ec650f3f60d40 (refs/changes/81/2581/2)
Author: Sean Chittenden <sean@chittenden.org>
Date:   2017-10-02T18:11:41-07:00 (2 years ago)
    
    Add useful contrib/ modules.  Surprise!, these are useful in production!!!

diff --git a/Makefile.postgres b/Makefile.postgres
index cdd2ecd..c02e935 100644
--- a/Makefile.postgres
+++ b/Makefile.postgres
@@ -82,9 +82,36 @@ pg96:
 	cd $(DEPSDIR)/postgresql96 && env \
 		CFLAGS=-m64 LDFLAGS=-m64 \
 		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/auto_explain && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/oid2name && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/pageinspect && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/pg_buffercache && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/pg_freespacemap && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/pg_prewarm && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
 	cd $(DEPSDIR)/postgresql96/contrib/pg_stat_statements && env \
 		CFLAGS=-m64 LDFLAGS=-m64 \
 		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/pg_visibility && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/pgstattuple && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
+	cd $(DEPSDIR)/postgresql96/contrib/vacuumlo && env \
+		CFLAGS=-m64 LDFLAGS=-m64 \
+		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
 	cd $(DEPSDIR)/pg_repack && env \
 		CFLAGS=-m64 \
 		PATH=$(RELSTAGEDIR)/root/$(PG96DIR)/bin:$$PATH \
